cli/cmd/uninstall.go

```diff
@@ -143,6 +151,25 @@ func UninstallClusterResources(ctx context.Context, client *kube.Client, ns stri
 
 }
 
+func waitForPodsToRolloutWithoutInstrumentation(ctx context.Context, client *kube.Client) {
+	l := log.Print("Rolling back odigos changes to pods...")
```

Comment:
suggestion: we are going to preform an operation that can take 5 min.
Let's say something about it in the message so user knows cli is not stuck